### ISSUE TITLE
Currently Using Float Left Instead Of Flex Grid

### ISSUE THREAD

**Issue by RianBergen** [RianBergen/RB-Blog-Template#1]
## Description

I am currently using float left instead of a flex grid for my Popular Posts List (Images).
This should be replaced with a Flex Grid!

### Code

```HTML
            <!-- START - Popular Posts -->
            <div class="rb-card">
                <div>
                    <h3><b>Popular Posts</b></h3>
                </div>

                <hr/>

                <div class="rb-list">
                    <a href="#0" class="rb-button rb-list-item">
                        <img src="images/Placeholder-Image-128x128.png" alt="Placeholder Image">
                        <span class="rb-text-font-large">Lorem</span><br>
                        <span>ipsum dolor sit</span>
                    </a>
                    <a href="#0" class="rb-button rb-list-item">
                        <img src="images/Placeholder-Image-128x128.png" alt="Placeholder Image">
                        <span class="rb-text-font-large">Lorem</span><br>
                        <span>ipsum dolor sit</span>
                    </a>
                    <a href="#0" class="rb-button rb-list-item">
                        <img src="images/Placeholder-Image-128x128.png" alt="Placeholder Image">
                        <span class="rb-text-font-large">Lorem</span><br>
                        <span>ipsum dolor sit</span>
                    </a>
                    <a href="#0" class="rb-button rb-list-item">
                        <img src="images/Placeholder-Image-128x128.png" alt="Placeholder Image">
                        <span class="rb-text-font-large">Lorem</span><br>
                        <span>ipsum dolor sit</span>
                    </a>
                </div>
            </div>
            <!-- END   - Popular Posts -->
```

```CSS
.rb-list-item > img {
    width: 3.125rem;
    margin-right: 1rem !important;
    float: left;    /* I HATE Using Float In This Instance, But I Wasn't Able To6 Get Flex Box To Do What I Wanted */
}
```

**Comment by GerritBergen** [comment]
Fixed with pull request #2

**Comment by RianBergen** [comment]
Thanks!
